src/test/java/com/bakdata/common_kafka_streams/test_applications/Mirror.java

```diff
@@ -33,12 +33,12 @@
 public class Mirror extends KafkaStreamsApplication {
     @Override
     public void buildTopology(final StreamsBuilder builder) {
-        final KStream<String, String> input = builder.stream(this.getInputTopic());
+        final KStream<String, String> input = builder.stream(this.getInputTopics());
```

Comment:
While this may be correct in this specific use case, it is an unintended way of using the Kafka API. When passing a Collection to builder.stream, all input topics are simply merged into a single stream without ordering guarantees. Please use this.inputTopics.get(0) to make it more explicit here.